### ISSUE TITLE
Show buckets a drop has been added to

### ISSUE THREAD

**Issue by dan-king** [ushahidi/SwiftRiver#253]
It would be nice if the "list" and "drops" pages showed the buckets that each drop already belongs to. At the moment you have to manually open each drop to see what bucket, if any, it belongs to.

**Comment by ekala** [comment]
As a note, a drop can belong to more than one bucket. Perhaps a visual distinction (different background or border color etc) to denote that a drop is already in a bucket?

**Comment by dan-king** [comment]
Thanks for the clarification. Yes a visual distinction such as bg color would work to show that someone looked at it. That's the main thing. A bonus would be a badge in top-right of drop showing the count of buckets. Maybe overkill but just a thought.

**Comment by 69mb** [comment]
+1 on having a different background for drops already in a bucket.

@brandonrosage something similar came up some time back for read/unread drops but this should be easier to implement.

**Comment by brandonrosage** [comment]
[In my latest commit in the "ui-dev" branch](https://github.com/ushahidi/SwiftRiver/commit/248e14cf364f6c608c67203fc3df5c042bf4b674), I've revised drops' indicator for the number of buckets it belongs to. [The indicator extends the width of the "Add to bucket" button](http://cl.ly/image/2z2C2Y2u2d0l) to display the total.

[I've added the frontend markup to the application code](https://github.com/ushahidi/SwiftRiver/blob/ui-dev/themes/default/views/pages/drop/drops.php#L33), commented out.

I want to tread carefully with changes to drops' background color for a few reasons:
- It's plausible for a drop to appear in a new river that you've encountered before and have already added to a bucket. In this case, it'd already be visually altered -- too dramatically, I think.
- There's some competition from, say, "buoys" (or "bookmarks") in the need to alter the visual state of an entire drop to communicate something. And in that case, I think changing the visual state of an entire drop is more appropriate for orienting someone on the page than indicating that it's been added to a bucket.
